### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Tommymarvel**
## YOUTH CONFERENCE 
# This PR consist of an HTML file, CSS, and javascript for object
- Here is the loom video for my project https://www.loom.com/share/e900237c9a4a43b7ba2f99c97702e115

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" />
+  <title>About session</title>
+</head>
```
Comment (ishimwezachee) [inline about.html:15]: - [ ]  You should have the same navigation links on your about pages, please fix this issue to go to the next milestone 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            one that leads to our future. Our youth our tomorrow.
+          </p>
+          <h2 class="event-date">
+            2022 <b>.</b> 09 <b>.</b> 23 (Fri) ~ 24(Sat)
+          </h2>
+          <p class="event-address">
+            @ Adebayo Akande Hall, University of Ibadan
+          </p>
+        </div>
+      </section>
```
Comment (ishimwezachee) [inline index.html]: - [ ] You should have navigations similar to the one from the designs 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        </div>
+
+        <div class="content">
+          <img class="pics" src="./images/exhibition.jpg" alt="" />
+          <p class="event-topics">Exhibition</p>
+          <p class="topic-info">
+            Display of creative youth projects that can serve as examples to
+            other
+          </p>
+        </div>
```
Comment (ishimwezachee) [inline index.html]: You should put you image s in the centers 
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Tommymarvel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
The project is running successfully 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (RolandM99) [reply about.html:15]: - [ ] As requested by the previous reviewer, make sure that you're using the same navigation for the desktop version on the about page. It seems that you forgot to implement your menu as expected for the desktop version for `about` page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+      <div class="nav2">
+        
+        <ul class="nav2-content">
+          <li class="nav2logo"></li>
+          <li class="nav2list"><a href="./index.html">Home</a></li>
+          <li class="nav2list"><a href="./about.html">About</a></li>
+          <li class="nav2list"><a href="#sponsors">Sponsors</a></li>
+          <li class="nav2list"><a href="#more-info">Contact us</a></li>
+        </ul>
+      </div>
```
Comment (RolandM99) [inline index.html]: - [ ] For this project you should follow exactly the original design from Behance. As requested by the previous reviewer, Kindly make sure that your navigation design fit the original design from Behance. here is the original design:
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Tommymarvel, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [ OPTIONAL ] Kindly, make sure that you added your `node_modules` directory in your `.gitignore` file. This will help you to ignore all the packages inside the subdirectory `node_modules` and use only the packages you need.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (Whoistolu) [reply about.html:15]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Tommymarvel 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [OPTIONAL] Kindly ensure that the `node_modules` directory is properly ignored by the `.gitignore` file. The `node_modules` folder is where all packages required by your JavaScript project are downloaded and installed. This folder is commonly excluded from a remote repository because it has a large size and you shouldn’t add code that you didn’t write to the repository.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**